vladimir.ohantsev/task-2-1/cmd/service/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/P3rCh1/task-2-1/internal/department"
+)
+
+func scanWorkerRequest() (*department.ChangeRequest, error) {
+	req := new(department.ChangeRequest)
+	if _, err := fmt.Scan(&req.Operator); err != nil {
+		return nil, fmt.Errorf("failed to scan operator: %w", err)
+	}
+
+	if _, err := fmt.Scan(&req.Temperature); err != nil {
+		return nil, fmt.Errorf("failed to scan temperature: %w", err)
+	}
+
+	return req, nil
+}
+
+func processDepartment() error {
+	var countWorkers int
+	if _, err := fmt.Scan(&countWorkers); err != nil {
+		return fmt.Errorf("failed to scan count workers: %w", err)
+	}
+
+	dep := department.New()
+
+	for range countWorkers {
+		req, err := scanWorkerRequest()
+		if err != nil {
+			return fmt.Errorf("process worker fail: %w", err)
```

Comment:
<img width="400" height="468" alt="Image" src="https://github.com/user-attachments/assets/1bc41f59-c8b0-4efd-b06c-78fff92b1524" />

ну в целом и так понятно, что `fail`


```suggestion
			return fmt.Errorf("process worker: %w", err)
```